Modules/_ssl.c

```diff
@@ -6165,7 +6153,6 @@ sslmodule_traverse(PyObject *m, visitproc visit, void *arg)
     Py_VISIT(state->PySSLSyscallErrorObject);
     Py_VISIT(state->PySSLEOFErrorObject);
     Py_VISIT(state->err_codes_to_names);
-    Py_VISIT(state->err_names_to_codes);
     Py_VISIT(state->lib_codes_to_names);
```

Comment:
I'm not very familiar with the CPython API, so I wasn't sure if this should be removed now that the objects are no longer exported. I noticed `Sock_Type` doesn't seem to be exported, but is traversed, while `str_library` is also unexported but not traversed.